src/FocusApp.Client/Helpers/PopupService.cs

```diff
@@ -0,0 +1,50 @@
+﻿using CommunityToolkit.Maui.Views;
+
+namespace FocusApp.Client.Helpers
+{
+    public interface IPopupService
+    {
+        void ShowPopup<T>() where T : Popup;
+        void HidePopup();
+        void HideAllPopups();
+    }
+
+    public class PopupService : IPopupService
+    {
+        private readonly IServiceProvider services;
+
+        // Need to keep track of the popups so we can close them.
+        private Stack<Popup> popups = new();
```

Comment:
Could we have these fields renamed to match the c# standards? (just have an _ at the beginning of the name, there are some examples in TimerService, but it would be _popups and _services.

Additionally, could we have both of them instantiated in the constructor? 
It helps with readability and maintainability

Reply:
I renamed the variables but what do you mean by instantiated in the constructor?